pkgs/top-level/all-packages.nix

```diff
@@ -25739,6 +25739,7 @@ in
   neovimUtils = callPackage ../applications/editors/neovim/utils.nix { };
   neovim = wrapNeovim neovim-unwrapped { };
 
+  neovim-qt-unwrapped = libsForQt5.callPackage ../applications/editors/neovim/neovim-qt.nix { };
```

Comment:
```suggestion
  neovim-qt-unwrapped = libsForQt5.callPackage ../applications/editors/neovim/qt-unwrapped.nix { };
```

Reply:
I find "neovim-qt" clearer since that's the actual name of the program, we can imagine several qt neovim GUIs.